modules/mono/editor/bindings_generator.cpp

```diff
@@ -2914,13 +2916,13 @@ void BindingsGenerator::_populate_builtin_type_interfaces() {
 
 #define INSERT_ARRAY(m_type, m_proxy_t) INSERT_ARRAY_FULL(m_type, m_type, m_proxy_t)
 
-	INSERT_ARRAY(PackedIntArray, int);
+	INSERT_ARRAY(PackedInt32Array, int);
 	INSERT_ARRAY_FULL(PackedByteArray, PackedByteArray, byte);
 
 #ifdef REAL_T_IS_DOUBLE
-	INSERT_ARRAY(PackedRealArray, double);
+	INSERT_ARRAY(PackedFloat32Array, double);
 #else
-	INSERT_ARRAY(PackedRealArray, float);
+	INSERT_ARRAY(PackedFloat32Array, float);
```

Comment:
@neikeq That also looks bogus, but I guess a lot of changes are needed to the Mono bindings to take into account recent Variant changes anyway.

Reply:
To clarify on `REAL_T_IS_DOUBLE`, see https://github.com/godotengine/godot/pull/36515#issuecomment-590543561
I guess this might not be relevant to check here, we now explicit support both `float` and `double`. Variant's `float` is always a `double`, this won't be configurable. `real_t` will only be used in specific math code that should be float by default for performance, but can be upgraded to double for extra precision at the cost of memory.

Reply:
ah yes they are no longer related to real_t, they are just different types